frontends/web/src/routes/exchanges/exchanges.tsx

```diff
@@ -0,0 +1,237 @@
+                                    <FilterButton
+                                        active={methods.includes('CC')}
+                                        onClick={() => this.toggleMethods('CC')} >
+                                        {t('exchanges.methods.CC')}
+                                    </FilterButton>
+                                    <FilterButton
+                                        active={methods.includes('DCA')}
+                                        onClick={() => this.toggleMethods('DCA')} >
+                                        {t('exchanges.methods.DCA')}
+                                    </FilterButton>
+                                    <FilterButton
+                                        active={methods.includes('SW')}
+                                        onClick={() => this.toggleMethods('SW')} >
+                                        {t('exchanges.methods.SW')}
+                                    </FilterButton>
+                                    <FilterButton
+                                        active={methods.includes('P2P')}
+                                        onClick={() => this.toggleMethods('P2P')} >
+                                        {t('exchanges.methods.P2P')}
+                                    </FilterButton>
+                                </div>
+                            </div>
+                            <div className={styles.results}>
+                                {results.length ? results : t('exchanges.nomatch')}
+                            </div>
+                        </div>
+                        <Footer>
+                            <SwissMadeOpenSource />
+                        </Footer>
+                    </div>
+                </div>
+            </div>
+        );
+    }
+}
+
+interface FilterButtonProps {
+    active?: boolean;
+    onClick: () => void;
+    children: JSX.Element;
+}
+
+function FilterButton({
+    active = false,
+    onClick,
+    children,
+}: RenderableProps<FilterButtonProps>): JSX.Element {
+    return (
+        <Button
+            primary={active}
+            transparent={!active}
+            onClick={onClick} >
+            {children}
+        </Button>
+    );
+}
+
+function Row({
+    key,
+    description,
+    hostname,
+}: RenderableProps<Exchange>): JSX.Element {
+    return (
+        <A
+            key={key}
+            href={`https://s.shiftcrypto.ch/${key}`}
+            title={hostname}
+            className={styles.row}
+        >
+            <div className={styles.image}>
+                <img src={`/assets/exchanges/logos/${key}.png`} alt={key} />
+            </div>
+            <div className={styles.description}>
+                {description}
+                <div className="exchange-hostname">
+                    {hostname}
+                </div>
+            </div>
+            <div className={styles.button}>
+                <Button primary>Buy</Button>
```

Comment:
should this be `t('buy')` or something like that?

Reply:
yes, ... if we keep all the buy buttons.
and then I need to findout how to properly translate it there in tsx world, it's probably easy. :)

Reply:
Fixed in 2f39cf13565ece647608441711587cbd455d2f65